afanasev_nikita/prog.cpp

```diff
@@ -109,5 +124,93 @@ int main() {
   graph.add_edge(11, 13);
   graph.add_edge(12, 13);
 
+  return graph;
+}
+
+namespace printing {
+namespace json {
+std::string print_vertex(const Graph::Vertex& vertex, const Graph& graph) {
+  std::string result = "{";
+
+  result += "\"id\":";
+  result += std::to_string(vertex.id());
+
+  result += ",";
+
+  result += "\"edge_ids\":[";
+  const auto& temp = graph.connected_edge_ids(vertex.id());
+  for (const auto edge_id : temp) {
+    result += std::to_string(edge_id);
+    result += ",";
+  }
+  if (!result.empty()) {
+    result.pop_back();
+  }
+  result += "]";
+
+  result += "}";
+  return result;
+}
+std::string print_edge(const Graph::Edge& edge) {
+  std::string result = "{";
+
+  result += "\"id\":";
+  result += std::to_string(edge.id());
+
+  result += ",";
+
+  result += "\"vertex_ids\":[";
+  result += std::to_string(edge.from_vertex_id());
+  result += ",";
+  result += std::to_string(edge.to_vertex_id());
+  result += "]";
+
+  result += "}";
+  return result;
+}
+std::string print_graph(const Graph& graph) {
+  std::string result = "{";
+
+  result += "\"vertices\":[";
+  const auto& vertices = graph.vertices();
+  for (const auto& vertex : vertices) {

```

Comment:
Более удобная запись
```suggestion
  for (const auto& [vertex_id, vertex] : vertices) {
```